src/System.Linq.Queryable/src/System/Linq/Queryable.cs

```diff
@@ -39,10 +39,10 @@ public static IQueryable<TSource> Where<TSource>(this IQueryable<TSource> source
             return source.Provider.CreateQuery<TSource>(
                 Expression.Call(
                     null,
-                    GetMethodInfoOf(() => Queryable.Where(
+                    GetMethodInfoOf(() => Where(
```

Comment:
In another PR we have converged on delegate-based InfoOf. Supposedly the reducer can handle simple cases.